03_reseau/tp/part_3/chat_server.py

```diff
@@ -14,13 +14,17 @@ def _send_all(msg):
 def _handle_rclient(client_reader, client_writer):
     nickname = yield from client_reader.readline()
     nickname = nickname.decode('utf-8').strip()
+    if not nickname or not nickname.strip():
```

Comment:
Seems redundant, wouldn't `not nickname.strip()` be enough ? unless you think the overhead of running `strip()` on an empty string is too high ?